Client/Frontend/Settings/SearchBar/SearchBarSettingsViewModel.swift

```diff
@@ -59,10 +59,10 @@ final class SearchBarSettingsViewModel: FeatureFlaggable {
     weak var delegate: SearchBarPreferenceDelegate?
 
     private let prefs: Prefs
-    private let notificationCenter: NotificationCenter
-    init(prefs: Prefs, notificationCenter: NotificationCenter = NotificationCenter.default) {
+    private let notificationCenter: NotificationProtocol
+    init(prefs: Prefs, notificationProtocol: NotificationProtocol = NotificationCenter.default) {
         self.prefs = prefs
-        self.notificationCenter = notificationCenter
+        self.notificationProtocol = notificationProtocol
```

Comment:
The variable was renamed, but not `self.notificationProtocol`. You should make sure the build passes locally before pushing the changes.

Reply:
Yeah I tried to build but it is showing errors at `notificationCenter.post` in RustPlaces (8/10) and `NotificationCenter.default` in SwiftData (3/3).  
<img width="292" alt="pic" src="https://user-images.githubusercontent.com/114816569/227522256-197c95e7-ffca-41e1-8546-13532261df1f.png">

Should I change this code back to `self.notificationCenter = notificationCenter`?

Reply:
Getting these 3 errors after adding `anObject: nil` to `notificationCenter.post` at 6 places where I was getting error as "Missing argument for parameter 'anObject' in call". 

<img width="857" alt="Screenshot 2023-03-25 at 3 09 03 PM" src="https://user-images.githubusercontent.com/114816569/227712565-b67948fc-923b-4397-bc4c-5b83c0c14a5a.png">
<img width="1280" alt="Screenshot 2023-03-25 at 3 10 37 PM" src="https://user-images.githubusercontent.com/114816569/227712577-1e75516a-7f98-424f-99b4-a779ea941333.png">
<img width="1280" alt="Screenshot 2023-03-25 at 3 45 36 PM" src="https://user-images.githubusercontent.com/114816569/227712581-2da7d080-888c-4772-b2f0-61849ba6d811.png">

Reply:
Answered on Matrix on how to solve those errors.